go_sequencer/database/kvdb/kvdb.go

```diff
@@ -219,6 +222,7 @@ func (k *KVDB) reset(batchNum common.BatchNum, closeCurrent bool) error {
 		}
 		k.db = sto
 		k.CurrentAccountIdx = common.RollupConstReservedIDx // 255
+		k.CurrentVouchIdx = common.RollupConstReservedIDx
```

Comment:
@Shailu-s Is `CurrentVouchIdx` also reset as `RollupConstReservedIDx`(255) when kvdb is reset?